test/odt/native/listContinueNumbering.native

```diff
@@ -0,0 +1,89 @@
+[ OrderedList
```

Comment:
I may have missed it, but do you have a test where the first list has one item, containing a sublist with two items, and then a second list continues the first list? That kind of case would be good to test explicitly.

Reply:
Will create additional testcase.